src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -142,26 +199,22 @@ It must be installed and run under the name *up*, as referenced in the
 .. literalinclude:: ../../workflows/xtrigger/workflow_state/downstream/flow.cylc
```

Comment:
Any particular reason why these `flow.cylc` files are executable?

Reply:
Nope, news to me. I'll fix it.